### ISSUE TITLE
trouble in pmag_gui

### ISSUE THREAD

**Issue by ltauxe** [PmagPy/PmagPy#629]
**What command was run:**

pmag_gui.py -WD .
then clicked on 1. Convert magnetometer files....

**What happened:**

nothing!

**What I expected to happen:**
I expected the dialog gui to come up with options for file conversion

**Data file(s):**

**Screenshot(s):**

(base) ltauxe:$ pmag_gui.py -WD .
-I- Importing Pmag GUI dependencies
-I- Starting Pmag GUI - please be patient
-I- Read in any available data from working directory
-I- Using online data model
-I- Getting method codes from earthref.org
-I- Importing controlled vocabularies from https://earthref.org

then - nothing!
**PmagPy version:**
pmagpy-4.2.71

**Comment by njarboe** [comment]
The current master of pmag_gui.py works for me. Just pulled and then ran pmag_gui.py
Here is a screen shot of the terminal and the pop-up after converting one of Ken's files.
![Screen Shot 2021-03-11 at 2 39 21 PM](https://user-images.githubusercontent.com/1278789/110866373-e6d05a00-8279-11eb-8145-21818b928667.png)

**Comment by Swanson-Hysell** [comment]
I think that I fixed this issue with commit 177b4f8

In that commit, I commented out the last line of ```on_convert_file()``` function which had a ```self.hide()``` command  for some reason.

```
 def on_convert_file(self, event):
        pmag_dialogs_dia = pmag_gui_dialogs.import_magnetometer_data(self, wx.ID_ANY, '', self.WD)
        pmag_dialogs_dia.Show()
        pmag_dialogs_dia.Center()
#        self.Hide()
```

I will close the issue for now, but do reopen if issues are persisting.
